assets/components/directDebit/directDebitReducer.js

```diff
@@ -8,7 +8,7 @@ import type { Action } from './directDebitActions';
 
 export type DirectDebitState = {
   isPopUpOpen: boolean,
-  bankSortCode: string,
+  sortCode: Array<string>,
```

Comment:
I would keep this field as bankSortCode to math bankAccountNumber

Reply:
I think bankSortCode is needless repetition, all sort codes are bank sort codes. I'd actually be tempted to change bankAccountNumber to accountNumber as well.

Reply:
Agree, I would keep "bank" in both fields or drop it from both.